src/metabase/driver/generic_sql.clj

```diff
@@ -306,19 +306,25 @@
   [driver, ^DatabaseMetaData metadata]
   (let [all-schemas (set (map :table_schem (jdbc/result-set-seq (.getSchemas metadata))))
         schemas     (set/difference all-schemas (excluded-schemas driver))]
-    (set (for [schema     schemas
-               table-name (mapv :table_name (get-tables metadata schema))]
-           {:name   table-name
-            :schema schema}))))
+    (set (for [schema  schemas
+               table   (get-tables metadata schema)]
+           (merge {:name   (:table_name table)
+                   :schema schema}
+                  (let [remarks (:remarks table)]
+                    (when (not (str/blank? remarks))
```

Comment:
https://github.com/bbatsov/clojure-style-guide#when-not

Reply:
It should be fine to do 

```clojure
{:name        (:table_name table)
 :schema      schema
 :description (when-not (str/blank? remarks)
                remarks)}
```

since we should be checking whether `description` is `nil` anyway before saving it